server/src/lib/utils/error-parser.ts

```diff
@@ -37,11 +39,19 @@ function fixErrors(errorList: Obj, identifier: string): Obj {
     }
   }
 
+  if (failures) {
+    errorList.failed = errorList.failed + 1
```

Comment:
Can you please use shorthand form, in line 43, 45, 85, 87 and 241

Reply:
done @SarvaBharan